src/storage/blockchain/chainid.ts

```diff
@@ -0,0 +1,55 @@
+/**
+ * Object containing chain IDs for various blockchains and networks.
+ * @type { [key: string]: number }
+ */
+export const CHAIN_IDS: { [key: string]: number } = {
```

Comment:
please add current chain id, so we can avoid changes to this map, as long as we can

Reply:
Done. We can keep state in this global variable.